packages/core/src/common/props.ts

```diff
@@ -82,6 +82,7 @@ export interface ILinkProps {
     target?: string;
 }
 
+/** @deprecated use IControlledProps2 */
```

Comment:
please merge this into the following line, it should be one block comment. otherwise this deprecation is not associated with the symbol